presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/InternalResourceGroup.java

```diff
@@ -583,6 +621,11 @@ public InternalResourceGroup getOrCreateSubGroup(String name)
         }
     }
 
+    private boolean isSubResourceGroupStatic(Optional<ResourceGroupSelectionContext> resourceGroupSelectionContext)
```

Comment:
Not sure I follow this.  Isn't this just checking if the present RG is static?  If so, why the name?

Reply:
This is actually checking if the sub resource groups is dynamic.